### PR TITLE
add issuing pallet

### PR DESCRIPTION
**Pull request by xiaoch05**
Add a Issuing pallet here to issue native asset from substrate<>substrate parachain bridge. Such as `pangolin<>pangolin parachain` or `crab<>crab parachain`.
Also users can burn their native asset on parachain to redeem the original token on the source chain.

### CODE REVIEW

**Comment by acgxv** [comment]
I don't have too much knowledge to review this.
Please add some description.

File: runtime/common/src/helixbridge/mod.rs [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+		/// The remote chain name where the backing module in
+		type BackingChainName: Get<ChainName>;
+
+		/// The lane id of the s2s bridge
+		type MessageLaneId: Get<LaneId>;
+
+		/// The local token decimals.
+		#[pallet::constant]
+		type DecimalsDifference: Get<RingBalance<Self>>;
```
Comment (hackfisher) [inline runtime/common/src/helixbridge/mod.rs]:
```suggestion
		type DecimalMultiplier: Get<u128>;
```
I'm not sure this multiplier's type should be `RingBalance ` or not.

I think it should be u128

**Review by hackfisher — COMMENTED** [review]
Generally LGTM.

File: runtime/common/src/helixbridge/mod.rs (hunk 1 above)
Comment (xiaoch05) [reply runtime/common/src/helixbridge/mod.rs]: Defining decimals using u128 is more accurate, but the operations between Balance and u128 still require a type conversion. So I'm not sure which type is better.

**Comment by hackfisher** [comment]
@xiaoch05 CI failed.

**Review by hackfisher — APPROVED** [review]

**Pull request merged**